docs/using-hive.md

```diff
@@ -16,6 +16,32 @@
         $ oc create secret generic test-creds --from-literal=aws_secret_access_key=$AWS_SECRET_ACCESS_KEY --from-literal=aws_access_key_id=$AWS_ACCESS_KEY_ID
         ```
 
+### Using Global Pull Secret
+
+GlobalPullSecret can be used to specify a pull secret that will be used globally by all of the cluster deployments created by Hive. 
+When GlobalPullSecret is defined in Hive namespace and a cluster deployment specific pull secret (local pull secret) is sepcified, the contents of both pull secrets will be merged and then will be used for provisioning the OpenShift cluster. 
```

Comment:
"sepcified" typo.

Suggest: "the registry authentication in both secrets will be merged and used by the new OpenShift cluster". (as it is used permanently by the cluster, not just for provisioning)

Reply:
I'd rather "local pull secret" didn't make it to the docs as the term doesn't seem quite right for user facing. Maybe "cluster specific pull secret"?